src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int maxRandomValue = 100;
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
        private final Random random = new Random();
        private final ColorSupplier colorSupplier = new ColorSupplier();
```
and move them on the next line after class